docs/sources/tempo/configuration/parquet.md

```diff
@@ -22,14 +22,16 @@ Block formats based on Parquet require more CPU and memory resources than the pr
 
 ## Choose a different block format
 
-The default block format is `vParquet2` which is the latest iteration of Tempo's Parquet based columnar block format.
+The default block format is `vParquet2`, which is the latest iteration of Tempo's Parquet based columnar block format.
 It is still possible to use the previous format `vParquet`.
 To enable it, set the block version option to `vParquet` in the Storage section of the configuration file.
 
+<!-- Commenting out until vParquet3 is released.
```

Comment:
may as well just delete since this is the 2.2 notes, but not a big deal either way

Reply:
At least with having it here, when we make vParquet3 available, we have the content instead of recreating it.